src/os/file_anyos.go

```diff
@@ -147,6 +147,23 @@ func Chmod(name string, mode FileMode) error {
 	return nil
 }
 
+// Chown changes the numeric uid and gid of the named file.
+// If the file is a symbolic link, it changes the uid and gid of the link's target.
+// A uid or gid of -1 means to not change that value.
+// If there is an error, it will be of type *PathError.
+//
+// On Windows or Plan 9, Chown always returns the syscall.EWINDOWS or
+// EPLAN9 error, wrapped in *PathError.
```

Comment:
TinyGo doesn't support Plan 9, so I'm not sure how useful this comment is?

Reply:
You're right, this should be removed.